templates/nodejs/.filters/all.js

```diff
@@ -192,4 +192,55 @@ module.exports = ({ Nunjucks, _ }) => {
   Nunjucks.addFilter('filenamify', (string, options) => {
     return filenamify(string, options || { replacement: '-', maxLength: 255 });
   });
+
+  /**
+   * Replaces variables in the server url with its declared values. Default or first enum in case of default is not declared
+   * Replace is performed only if there are variables in the URL and they are declared for a server
+   * @private
+   * @param {String} url The server url value.
+   * @param {Object} serverVariables object containins server variables.
+   * @return {String}
+   */
+  function replaceVariablesWithValues(url, serverVariables) {
+    const urlVariables = getVariablesNamesFromUrl(url);
+    const isVariableDeclared =
+      urlVariables.filter(el => serverVariables.hasOwnProperty(el[1])) !== 0;
```

Comment:
Did you mean this?

```suggestion
      urlVariables.filter(el => serverVariables.hasOwnProperty(el[1])).length !== 0;
```

Reply:
nope, but it is still a bug, thanks for pointing out. `hasOwnProperty` returns Boolean. I fixed it and renamed the variable to reflect the reality